src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> storage = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<Fruit, Integer> storage = new HashMap<>();
```
чуть более гибко будет сделать тут Fruit c полем name
если добавятся другие поля, то не придется потом рефакторить код